client/src/components/SubmissionTable.vue

```diff
@@ -0,0 +1,162 @@
+    :rows="tableData"
+    row-key="id"
+    :filter="status_filter"
+    dense
+  >
+    <template #top>
+      <div class="row full-width justify-between q-pb-md">
+        <div class="column">
+          <h3 class="q-my-none">{{ $t(title) }}</h3>
+          <!-- eslint-disable-next-line vue/no-v-html -->
+          <p class="q-mb-none" v-html="$t(byline)"></p>
+        </div>
+        <div class="column">
+          <q-select
+            v-model="status_filter"
+            clearable
+            square
+            outlined
+            dense
+            :label="$t(`submission_tables.filter_label`)"
+            :options="unique_statuses"
+            style="width: 240px"
+            class="q-mt-md q-mr-xs"
+          >
+            <template #selected-item="scope">
+              {{ $t(`submission.status.${scope.opt}`) }}
+            </template>
+            <template #option="scope">
+              <q-item v-bind="scope.itemProps">
+                <q-item-section>
+                  <q-item-label>
+                    {{ $t(`submission.status.${scope.opt}`) }}
+                  </q-item-label>
+                </q-item-section>
+              </q-item>
+            </template>
+          </q-select>
+        </div>
+      </div>
+    </template>
+    <template #no-data>
+      <div class="full-width row flex-center text--grey q-py-lg">
+        <p class="text-h3">{{ $t(noData) }}</p>
+      </div>
+    </template>
+    <template #body="props">
+      <q-tr :props="props">
+        <q-td key="id" :props="props">
+          {{ props.row.id }}
+        </q-td>
+        <q-td key="title" :props="props">
+          <router-link
+            :to="{
+              name: 'submission_review',
+              params: { id: props.row.id },
+            }"
+            >{{ props.row.title }}
+          </router-link>
+        </q-td>
+        <q-td key="publication" :props="props">
+          <router-link
+            :to="{
+              name: 'publication:home',
+              params: { id: props.row.publication.id },
+            }"
+            >{{ props.row.publication.name }}
+          </router-link>
+        </q-td>
+        <q-td key="status" :props="props">
+          {{ $t(`submission.status.${props.row.status}`) }}
+        </q-td>
+        <q-td key="actions" :props="props">
+          <submission-table-actions
+            :submission="props.row"
+            action-type="reviews"
+            flat
+          />
+        </q-td>
+      </q-tr>
+    </template>
+  </q-table>
+</template>
+<script setup>
+import SubmissionTableActions from "./SubmissionTableActions.vue"
+import { useI18n } from "vue-i18n"
+import { ref, computed } from "vue"
+const { t } = useI18n()
+
+const propsVar = defineProps({
+  tableData: {
+    type: Array,
+    default: () => [],
+  },
+  title: {
+    type: String,
+    default: "",
+  },
+  byline: {
+    type: String,
+    default: "",
+  },
+  tableType: {
+    type: String,
+    default: "",
+  },
```

Comment:
Good spot. I meant to utilize this prop to easily toggle between review-specific actions and submission-specific actions for the nested `SubmissionTableActions` component because I'm anticipating `SubmissionTable` being re-used when we implement the new design on the submissions page. I'll fix this so it's no longer useless.